user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_SRM_Framework_Configuration.md

```diff
@@ -0,0 +1,55 @@
+---
+uid: BPA_SRM_Framework_Configuration
+---
+
+# SRM Framework Configuration
+
+This BPA test checks for Bookings with invalid Properties that have end date in the future.
+
+> [!NOTE]
+> The BPA can take some time to run, depending on the amount of bookings in the system.
+## Metadata
+
+* Name: SRM Framework Configuration
+* Description: Detects with invalid SRM Solution configurations.
+* Author: Skyline Communications
+* Default schedule: Daily
+
+## Results
+
+### Success
+
+* "No invalid Bookings detected in the system.*
+
+No inconsistencies have been detected in the system.
+
+### Error
```

Comment:
@JorgeFilipeDias, I notice that these error messages are different from the ones you sent me for review via email earlier, which seems a bit odd. Are these additional messages that still need to be reviewed?

Reply:
The ones in the mail were the actual code, the messages that the user will see.
Here we have a complete list of the properties with the explanation of the error.
E.g.: For '"Booking doesn't have Property with names: 'Booking Manager', 'Booking Life Cycle', 'FriendlyReference', 'PreRoll', 'PostRoll', 'Service State', 'Start', 'End', 'VisualBackground' and 'VisualForeground'"' what the user will see is ': Booking ID: 8cf046c2-a41b-4bbc-bbe2-7c53eff8fe03 Name: Example Booking does not contain ‘Start’ property'